<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>730</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGraphicsView" name="gameView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="beerLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>610</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P1 Beers left :</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="beerAmountLcd">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>610</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="staminaLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P1 Stamina:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="staminaLcd">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>610</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getOnButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>610</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get on bus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getOffButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>610</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get out of bus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drinkBeerButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>610</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Drink beer</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameStatusLabel">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>630</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>630</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>100</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="staminaLabel2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P2 Stamina:</string>
    </property>
   </widget>
   <widget class="QLabel" name="beerLabel2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>650</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P2 Beers left:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getOnButton2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>650</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get on bus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getOffButton2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>650</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get out of bus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drinkBeerButton2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>650</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Drink beer</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="staminaLcd2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>650</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="beerAmountLcd2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>650</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="scoreTextBrowser">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>370</y>
      <width>161</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>350</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name --- points</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>60</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Player name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1316</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
